attester/Cargo.toml

```diff
@@ -15,11 +15,14 @@ serde.workspace = true
 serde_json.workspace = true
 strum.workspace = true
 tdx-attest-rs = { git = "https://github.com/intel/SGXDataCenterAttestationPrimitives", rev = "cc582e8be0c9010295c66fb58c59f74744017600", optional = true }
+sev = { git = "https://github.com/virtee/sev", rev = "3dca05d2c93388cb00534ad18f5928fd812e99cc", optional = true }
+raw-cpuid = { version = "10.6.0", optional = true }
```

Comment:
where is `raw-cpuid` used? if not needed, remove here and in line 28.

Reply:
Ah yeah I think that is left over from an earlier approach for checking the platform type.